op-chain-ops/cmd/check-prestate/diff-check.zsh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env zsh
+
+cat - | jq -r '."outdated-chains"[] | .name + "\n" + .diff.message + "\n" + (.diff.prestate | tostring) + "\n" + (.diff.latest | tostring)' | while read -r name; do
+    read -r message
+    read -r prestate
+    read -r latest
+
+    echo "\n=== $name ===\n$message\n"
+
+    diff -u --label=prestate --label=latest <(echo "$prestate" | jq) <(echo "$latest" | jq)
```

Comment:
I think if you reverse the diff order here, it will be clearer.  Would expect the diff to be from the perspective of the prestate.

Reply:
I actually find the current order more intuitive, because we go _from_ the given prestate _to_ the latest configs. So latest is usually newer.

But I don't have a strong opinion here.

Reply:
yeah - not a big deal either way.  that was just my initial expectation.